worlds/sa2b/__init__.py

```diff
@@ -237,17 +261,59 @@ def generate_basic(self):
             musiclist_o = list(range(0, 47))
             musiclist_s = musiclist_o.copy()
             self.multiworld.random.shuffle(musiclist_s)
+            musiclist_o.extend(list(range(47, 78)))
+            musiclist_s.extend(list(range(47, 78)))
```

Comment:
```suggestion
            musiclist_o.extend(range(47, 78))
            musiclist_s.extend(range(47, 78))
```
extend does not appear to need a list call:

![image](https://user-images.githubusercontent.com/3189725/205857127-600aaa8a-5456-4448-9c5d-7fc71c6d8c5b.png)